### PR TITLE
Fix overactive HPET rtc interrupt.

### PR DESCRIPTION
**Pull request by gsilvis**
This decreases the compute load of the kernel significantly!

### CODE REVIEW

File: kernel/hpet.c [hunk 1]
```diff
@@ -102,11 +101,17 @@ void hpet_sleepers_awake() {
...
-    } else {
-      struct sleeper *s = (struct sleeper *)i;
-      current_deadline = s->deadline;
+      /* Nothing to wait for; set the timer far in the future.  According to
+       * the spec, 0xFFFFFFFFFFFFFFFF is a valid (and default) value for this
+       * timer, but in QEMU, setting the timer abover 0x0CCCCCCCCC222221L
+       * appears to cause an interrupt to happen immediately.  Use a value that
+       * is safely lower than that but also so far off into the future that
+       * we're not worried about running into it. */
+      hpet_reg->timers[1].comparator = 0x00FFFFFFFFFFFFFF;
```
Comment (allanlw) [inline kernel/hpet.c]: Why not just set to 0..?

Comment (gsilvis) [inline kernel/hpet.c]: Doesn't seem to work.

Who knows

**Comment by gsilvis** [comment]
I figured out the non-hacky solution: don't write /anything/ to the register.  Each write can only ever cause one interrupt, so there's no way to get tons of interrupts this way.

**Comment by allanlw** [comment]
Ouch. LGTM

**Pull request merged**